src/presto_api.rs

```diff
@@ -43,10 +44,11 @@ impl PrestoApi {
     ) -> Result<Request<Body>, Error> {
         let uri_str = format!("{}/v1/statement", base_url);
 
-        let request = Request::builder()
-            .method(Method::POST)
-            .uri(uri_str)
-            .header("X-Trino-User", "jagill")
+        let builder = Request::builder().method(Method::POST).uri(uri_str);
+
+        let request = HeaderBuilder::new()
```

Comment:
This is kind of a funny flow -- you instantiate a HeaderBuilder and then end up with a Request?  But we can work out those details later.